Makefile.tests

```diff
@@ -1215,5 +1224,5 @@ test: build testlib tests-common $(TESTS) \
 
 run-%:	$*
 	@echo running $*
-	@cd $(BUILDDIR) && CAML_LD_LIBRARY_PATH=. LD_LIBRARY_PATH=clib DYLD_LIBRARY_PATH=clib ./$*.$(BEST) -runner sequential
+	@cd $(BUILDDIR) && CAML_LD_LIBRARY_PATH=. LD_LIBRARY_PATH=clib DYLD_LIBRARY_PATH=clib ./$*.$(BEST) -runner sequential #$($*.test_args)
```

Comment:
This line appears to be unchanged, except for a commented-out part.  Can it be reverted?